DataDog/documentation
# search rum action

### What does this PR do?
Sends a custom RUM action based on user searches

### Motivation
More data on how users are searching is useful

### Preview 
https://docs-staging.datadoghq.com/brian.deutsch/search-rum-action

### Testing
- search from the homepage by either clicking `Submit` or pressing enter.
- search from the side nav on any page including clicking through one of the links in the dropdown
- click through one of the links from the search page
- custom rum actions should be sent to Datadog.   Try using the RUM actions tab in the explorer here: https://dd-corpsite.datadoghq.com/rum/explorer?query=%40application.id%3A3493b4e7-ab12-4852-8836-ba96af7bc745%20env%3Apreview%20%40session.type%3Auser%20%40action.type%3Acustom%20%40type%3Aaction&cols=&viz=stream&from_ts=1682261659467&to_ts=1682348059467&live=true to see the custom action being sent.

- there should be no console errors present and other UI javascript functionality is unaffected.  Please make sure to QA some JS elements (language/site dropdowns, mobile menu, TOC, tabs, etc)
- search should continue work including from the JA and FR sites and RUM actions should be posted from the translated sites as well.

### Additional Notes

---

### Reviewer checklist
